br/pkg/task/backup.go

```diff
@@ -301,6 +338,13 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 		return errors.Trace(err)
 	}
 	defer mgr.Close()
+	// after version check, check the cluster whether support checkpoint mode
+	if cfg.UseCheckpoint {
```

Comment:
shall we fall back to normal backup without checkpoint?

Reply:
And it looks like the default value of UseCheckpoint is true. then it must failed when run backup on v6.4.0?

Reply:
done, print the error to log, and fall back to normal backup mode